http/src/main/java/io/micronaut/http/ssl/ClientSslConfiguration.java

```diff
@@ -61,6 +65,13 @@ public ClientSslConfiguration() {
         setEnabled(true);
     }
 
+    public boolean isDisableHostnameVerification() {
+        return disableHostnameVerification;
+    }
+
+    public void setDisableHostnameVerification(boolean disableHostnameVerification) {
+        this.disableHostnameVerification = disableHostnameVerification;
+    }
```

Comment:
add missing javadoc with `@since` annotation mentioning the version this was added. We also need to document the consequence of this setting since this opens up a security risk